### DISCUSSION TITLE
Logging in to the co-pilot extension, giving ""No access to GitHub Copilot found."

### DISCUSSION THREAD

**Discussion by sumaini-cisco** [community/community#73202] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Debug logs:

[INFO] [auth] [2023-11-02T06:02:28.785Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-11-02T06:02:28.787Z] Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as xxxxxxx-xxxxx."
[INFO] [auth] [2023-11-02T06:33:44.430Z] Invalid copilot token: missing token: 403 
[INFO] [auth] [2023-11-02T07:21:07.605Z] Invalid copilot token: missing token: 403

**Comment by Akash1134** [comment]
Hi @sumaini-cisco ,

I would recommend that you signing out of your GitHub account in IDE, and signing in again with the GitHub account that has a Copilot subscription.

Thanks!

**Comment by davecheney** [comment, marked as answer]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Reply by clb168** [reply]
This solution is not working for me, is there anything else I can try?

**Reply by fabrielg** [reply]
I love you man thanks 💙
